frontend/app/views/giraffe/contribute.scala.html

```diff
@@ -225,6 +230,30 @@ <h3 class="cvc__heading">American Express</h3>
             <p>
                 Please note that your support of the Guardian’s journalism does not constitute a charitable donation, as such your contribution is not eligible for Gift Aid in the UK nor a tax-deduction elsewhere.
             </p>
+        </span>
+
+        <span data-shown="aud">
+            <p>If you have any questions about contributing to the Guardian, please
+                <a href="mailto:contribution.support@@theguardian.com" class="support-thanks__link">contact us here</a>.
+            </p>
+            <p>
+                The ultimate owner of the Guardian is The Scott Trust Limited, whose role it is to secure the editorial and financial independence of the Guardian in perpetuity. Reader contributions support the Guardian’s journalism.
+            </p>
+        </span>
+        <span data-shown="usd">
+            <p>If you have any questions about contributing to the Guardian, please
+                <a href="mailto:contribution.support@@theguardian.com" class="support-thanks__link">contact us here</a>.
+            </p>
+            <p>
+                The ultimate owner of the Guardian is The Scott Trust Limited, whose role it is to secure the editorial and financial independence of the Guardian in perpetuity. Reader contributions support the Guardian’s journalism.
+            </p>
+            <p>
+                Please note that the Guardian is not a charity, will not use your support for charitable
```

Comment:
Could this be pushed into PageInfo as an additional attribute thus removing the duplication here?